src/base/language_support.c

```diff
@@ -2804,6 +2967,65 @@ lang_strcmp_utf8 (const LANG_COLLATION * lang_coll, const unsigned char *str1, c
   return lang_strmatch_utf8 (lang_coll, false, str1, size1, str2, size2, NULL, false, NULL);
 }
 
+static int
+lang_strcmp_utf8_ts (const LANG_COLLATION * lang_coll, const unsigned char *str1, int size1,
+		     const unsigned char *str2, int size2)
```

Comment:
The existing function lang_strcmp_utf8() wraps the lang_strmatch_utf8() function. 
How about keeping function calls for the same function structure?

Reply:
lang_strcmp_utf8() calls the lang_strmatch_utf8() but the trailing space is ignored.

Reply:
What I said is that it would be better for lang_strcmp_utf8_ts() to call lang_strmatch_utf8_ts() like as-is. 
but, this is not an important issue.

Reply:
It's fixed as refactoring codes.